.github/workflows/lint-python.yml

```diff
@@ -84,7 +84,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ inputs.python-version }}
+      - id: get_version
+        uses: scene-connect/actions/python-package-manager/get_package_version@use-project-linter-versions
```

Comment:
You can also use the relative path to the file in this case. I think that is easier then using the git repository, since it is in the same repo, and you also don't need to refer to tags/branches

Reply:
Paths are relative to the PR's working directory, not to the `actions` repository.

There's `$GITHUB_ACTION_PATH` but that's relative to the running `action.yml`, I'm not sure how it'd work within the _workflow_...

I'm gonna revert that relative path change and just leave them pointing to the git paths.

Reply:
Ah, crap. Cool. But thats weird though. It means you need to make 2 PR's. you can't leave `@use-project-linter-version` in there since that won't exist after you merged this pr.

Reply:
Yeh no I'm gonna tweak the version before merging and re-tag this as `@ v2` still since it's not a breaking change.